Dev/config/fontStyles.js

```diff
@@ -0,0 +1,92 @@
+//This class will contain all of the font styles that should be used through out the app. The default size & fontFamily will be
+//defined here.
+import colors from './colors';
+import { PixelRatio, Platform, StyleSheet } from 'react-native';
+
+//The default font family that'll be used throughout the app
+const fontFamily = (Platform.OS === "ios" ? "Arial" : "sans-serif-medium");
+
+//Sets the base font that'll be used throughout the app based on the size of the screen
+//Sets the font size
+let baseFontSize = 16;
+const pixelRatio = PixelRatio.get();
+if (pixelRatio < 3) {
+    baseFontSize = 14;
+}
+
+export default StyleSheet.create({
+
+    smallTextStyleBlack: {
+        fontFamily,
+        fontSize: baseFontSize * 0.75,
```

Comment:
I really like that. Great job!

Reply:
One small suggestion is that you can create consts for different font sizes (titleFont, subtitleFont, bodyFont, bodyFontSmaller, buttonFont etc..) inspired from the chart below. Another option is to simply create consts with name baseFontSize, smallFontSize, largeFontSize, etc. with the values you have defined below. This way you if you change their definition (for example *0.6 instead of 0.75) you only need to change that one constant definition.

This is a nit, so not a blocker for committing. 
![image](https://user-images.githubusercontent.com/7612136/65482683-b2cb6400-de4e-11e9-804b-3b2dd22ce67c.png)